styleguide/source/assets/js/resource-tabs.js

```diff
@@ -0,0 +1,42 @@
+$('*[data-resource]').click(function(){
```

Comment:
This can be simplified using the jQuery UI plugin like this:

```
$( ".ama__resource-tabs" ).tabs();
//Add the class 'open tab' to the link you want the click event to fire on. Add data-tab-index="2" to tell the event to target tab 2 
$('.ama__open-tab').click(function (event) {
  $('.ama__resource-tabs').tabs("option", "active", $(this).data("tab-index"));
});
```

Reply:
https://jqueryui.com/tabs/